src/collection-view.js

```diff
@@ -165,27 +165,38 @@ Marionette.CollectionView = Marionette.View.extend({
   reorder: function() {
     var children = this.children;
     var models = this._filteredSortedModels();
-    var modelsChanged = _.find(models, function(model) {
+    var modelsAdded = _.find(models, function(model) {
```

Comment:
should this be find or filter. seems like it should be a filter case (otherwise rename var please)